### PR TITLE
Refresh upgrade guide

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Refresh upgrade guide
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Comment by thomasnares** [comment]
I'm not statisfied with the way guides are splitted / redundant: 

- keeping-up-to-date/upgrade
- keeping-up-to-date/use-autoupgrade-module

I may join/move some blocks of documentation

File: basics/keeping-up-to-date/upgrade-module/upgrade-cli.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+2. Remove the old module's directory:
+
+`rm -rf modules/autoupgrade`
+
+### Step 2: Install the new autoupgrade module
+
+1. Download the latest version of the autoupgrade module and place it in the /modules directory:
+
+`curl -L  https://github.com/PrestaShop/autoupgrade/releases/download/v4.15.0/autoupgrade.zip -o modules/autoupgrade.zip && unzip modules/autoupgrade.zip`
```
Comment (Seb33300) [inline basics/keeping-up-to-date/upgrade-module/upgrade-cli.md]: Replace URL by `https://github.com/PrestaShop/autoupgrade/releases/latest/download/autoupgrade.zip`?

Comment (kpodemski) [reply basics/keeping-up-to-date/upgrade-module/upgrade-cli.md]: @Seb33300 thanks Seb, you're right, it'd be better

**Review by kpodemski — APPROVED** [review]
There's still some work to do, but I will merge it as is for now :)

**Pull request merged**